server/routes/updateGoal/reviewUpdateGoalView.ts

```diff
@@ -1,16 +1,16 @@
 import type { PrisonerSummary } from 'viewModels'
-import type { UpdateGoalForm } from 'forms'
+import type { UpdateGoalDto } from 'dto'
 
 export default class ReviewUpdateGoalView {
-  constructor(private readonly prisonerSummary: PrisonerSummary, private readonly updateGoalForm: UpdateGoalForm) {}
+  constructor(private readonly prisonerSummary: PrisonerSummary, private readonly updateGoalDto: UpdateGoalDto) {}
```

Comment:
I've changed this to pass the DTO into the view object, rather than the form
The reason for this is that the DTO contains a single field for the `targetCompletionDate` of type `Date`, whereas the form contains 4 optional fields (the date, or the dd, mm and yyyy fields - all of which are string)
The form to DTO mapper takes care of the logic re: which fields from the form to use to construct the `targetCompletionDate` `Date` instance; so I felt this was easier / safer to use, rather than having some scary logic in the nunjucks view to work out the correct date to render.